### PR TITLE
Rough draft of ember-angle-brackets codemod import(index.js file)

### PR DESCRIPTION
**Pull request by kevtran2**
Based off of https://github.com/ember-codemods/ember-angle-brackets-codemod/tree/master/transforms/angle-brackets, this import is an attempt to compile the transform(transform.js) and telemetry(invokable.js) modules into a single working index.js file compatible with the Intuita codemod-registry. Note that there are some CommonJS and ES6 differences between the code used in the Ember code base and our codemod registry. This is just the index.js file, the rest(config, readme, and test files will be added once index.js is complete).

### CODE REVIEW

File: ember/5/ember-angle-brackets-codemod/index.js [hunk 1]
```diff
@@ -0,0 +1,609 @@
+/**
+ * This code is based on a public codemod, which is subject to the original license terms.
+ * Original codemod: https://github.com/ember-codemods/ember-3x-codemods/blob/master/transforms/app-controller-router-props/index.js
```
Comment (hbjORbj) [inline ember/5/ember-angle-brackets-codemod/index.js:3]: wrong link

File: ember/5/ember-angle-brackets-codemod/index.js [hunk 2]
```diff
@@ -0,0 +1,609 @@
...
+const { getTelemetry } = require('ember-codemods-telemetry-helpers');
+const path = require('path');
+const fs = require('fs');
+
+
+// const transform = require('./transform');
+const recast = require('ember-template-recast');
+const logger = require('../../lib/logger');
+
+const KNOWN_HELPERS = require('./known-helpers');
```
Comment (hbjORbj) [inline ember/5/ember-angle-brackets-codemod/index.js:40]: Let's use ESM syntax.

the following is from GPT:

ECMAScript (ES) Modules (ESM) and CommonJS are two different systems for organizing and importing modules in JavaScript. Here's a comparison of the two:

### CommonJS (CJS) Modules:

1. **Usage**: CommonJS is the module system used in Node.js and was prevalent in the JavaScript ecosystem before the standardization of ES Modules.

2. **Syntax for Importing**:
   ```javascript
   const myModule = require('./myModule'); // Import
   ```

3. **Asynchronous Loading**: CommonJS modules are loaded synchronously, which means the module is loaded and processed before the next line of code is executed.

4. **Module Structure**: Each file is a module, and the variables and functions defined within the module are scoped to that module unless explicitly exported.

5. **Exports**: To export functionality, you use `module.exports` or `exports`.
   ```javascript
   // Exporting in CommonJS
   module.exports = { someFunction };
   ```

6. **Static Analysis**: The structure of CommonJS modules allows for straightforward static analysis to determine dependencies.

7. **Use Cases**: CommonJS is commonly used in server-side development with Node.js and is great for modularizing code in a synchronous environment.

### ECMAScript (ES) Modules (ESM):

1. **Usage**: ECMAScript Modules (ESM) is the standard module system for JavaScript, defined by the ECMAScript specification (ES6/ES2015).

2. **Syntax for Importing**:
   ```javascript
   import myModule from './myModule'; // Import
   ```

3. **Asynchronous Loading**: ESM modules are loaded asynchronously, which allows for better performance and scalability, especially in modern browsers.

4. **Module Structure**: ESM enforces strict mode by default, and variables and functions are scoped to the module unless explicitly exported using `export`.
   
5. **Exports**: To export functionality, you use `export`.
   ```javascript
   // Exporting in ESM
   export { someFunction };
   ```

6. **Static Analysis**: The asynchronous nature of ESM makes static analysis more complex compared to CommonJS.

7. **Use Cases**: ESM is becoming more prevalent, especially in front-end development and modern back-end frameworks. It's the standard for browser-based development and is now also supported in recent versions of Node.js.

In summary, CommonJS is synchronous and was widely used in Node.js environments before ES Modules became a standard. ES Modules (ESM) is asynchronous, supports both synchronous and asynchronous loading, and has become the standard for modular development in modern JavaScript, especially in the browser and newer versions of Node.js.

Comment (hbjORbj) [inline ember/5/ember-angle-brackets-codemod/index.js:40]: this file doesn't exist.

File: ember/5/ember-angle-brackets-codemod/index.js [hunk 3]
```diff
@@ -0,0 +1,609 @@
...
+ */
+
+const { getOptions: getCLIOptions } = require('codemod-cli');
+const { getTelemetry } = require('ember-codemods-telemetry-helpers');
+const path = require('path');
+const fs = require('fs');
+
+
+// const transform = require('./transform');
+const recast = require('ember-template-recast');
```
Comment (hbjORbj) [inline ember/5/ember-angle-brackets-codemod/index.js:37]: this file doesn't exist.

File: ember/5/ember-angle-brackets-codemod/index.js [hunk 4]
```diff
@@ -0,0 +1,609 @@
...
+	FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+	AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+	LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+	OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+	SOFTWARE.
+ * License URL: https://github.com/ember-codemods/ember-no-implicit-this-codemod/blob/master/LICENSE
+ */
+
+const { getOptions: getCLIOptions } = require('codemod-cli');
+const { getTelemetry } = require('ember-codemods-telemetry-helpers');
```
Comment (hbjORbj) [inline ember/5/ember-angle-brackets-codemod/index.js:31]: this module doesn't exist, does it?

File: ember/5/ember-angle-brackets-codemod/index.js [hunk 5]
```diff
@@ -0,0 +1,609 @@
...
+	IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+	FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+	AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+	LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+	OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+	SOFTWARE.
+ * License URL: https://github.com/ember-codemods/ember-no-implicit-this-codemod/blob/master/LICENSE
+ */
+
+const { getOptions: getCLIOptions } = require('codemod-cli');
```
Comment (hbjORbj) [inline ember/5/ember-angle-brackets-codemod/index.js:30]: this module doesn't exist, does it?

File: ember/5/ember-angle-brackets-codemod/index.js [hunk 6]
```diff
@@ -0,0 +1,609 @@
...
+const fs = require('fs');
+
+
+// const transform = require('./transform');
+const recast = require('ember-template-recast');
+const logger = require('../../lib/logger');
+
+const KNOWN_HELPERS = require('./known-helpers');
+const _EMPTY_STRING_ = `ANGLE_BRACKET_EMPTY_${Date.now()}`;
+const { builders: b } = recast;
```
Comment (hbjORbj) [inline ember/5/ember-angle-brackets-codemod/index.js:42]: recast is undefined

File: ember/5/ember-angle-brackets-codemod/index.js (hunk 1 above)
Comment (kevtran2) [reply ember/5/ember-angle-brackets-codemod/index.js:3]: Good catch, modified.

File: ember/5/ember-angle-brackets-codemod/index.js (hunk 2 above)
Comment (kevtran2) [reply ember/5/ember-angle-brackets-codemod/index.js:40]: noted

Comment (kevtran2) [reply ember/5/ember-angle-brackets-codemod/index.js:40]: good catch, found file

File: ember/5/ember-angle-brackets-codemod/index.js [hunk 7]
```diff
@@ -0,0 +1,609 @@
...
+
+const { getOptions: getCLIOptions } = require('codemod-cli');
+const { getTelemetry } = require('ember-codemods-telemetry-helpers');
+const path = require('path');
+const fs = require('fs');
+
+
+// const transform = require('./transform');
+const recast = require('ember-template-recast');
+const logger = require('../../lib/logger');
```
[the remark this answers is not in the document]
Comment (kevtran2) [reply ember/5/ember-angle-brackets-codemod/index.js:38]: good catch found file

File: ember/5/ember-angle-brackets-codemod/index.js (hunk 3 above)
Comment (kevtran2) [reply ember/5/ember-angle-brackets-codemod/index.js:37]: I don't see this one

File: ember/5/ember-angle-brackets-codemod/index.js (hunk 4 above)
Comment (kevtran2) [reply ember/5/ember-angle-brackets-codemod/index.js:31]: I don't see where this module is in their repo...

File: ember/5/ember-angle-brackets-codemod/index.js (hunk 2 above)
Comment (kevtran2) [reply ember/5/ember-angle-brackets-codemod/index.js:40]: converted
